projects/timer_challenge/src/main.c

```diff
@@ -0,0 +1,41 @@
+// Firmware 102 Homework
+// Vaaranan Yogalingam
+// 2021-05-29
+
+#include <stdint.h>
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+#define PERIOD 500
```

Comment:
I'm not sure I like this naming here. Though it makes sense to me (since I know what the tasks of this hwk assignment is) it would be unclear to someone else.

What is the units for PERIOD? What is this PERIOD for? It would be best to change the naming so that it is more descriptive.

Normally in our code, we will add a suffix to the variable to describe the units eg:
DELAY_TIME_MS